src/components/dls/QuranWord/QuranWord.tsx

```diff
@@ -7,11 +7,11 @@ import useSWRImmutable from 'swr/immutable';
 import GlyphWord from './GlyphWord';
 import onQuranWordClick from './onQuranWordClick';
 import styles from './QuranWord.module.scss';
+import TajweedWord from './TajweedWordImage';
```

Comment:
`TajweedWordImage` will have to be renamed when we implement the Tajweed text instead of images so I would rather leave it as it is.